doc/environment.yaml

```diff
@@ -5,7 +5,7 @@ dependencies:
   - pip
   - pip:
     - -e ..
-    - ax-platform >= 0.4.0
+    - ax-platform == 0.4.0
```

Comment:
This is a temporary fix. I will open a new PR to make `optimas` compatible with `0.4.1`